### PR TITLE
Feature/components

### PR DESCRIPTION
**Pull request by MahdiAghaali**
In this branch I did the following activities and changes:

- Create the base for the calculator app
- Create Display, Button, and Calculator components
- Add some styling

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <Button displayText="7" type="digit" />
+    <Button displayText="8" type="digit" />
+    <Button displayText="9" type="digit" />
+    <Button displayText="/" type="operator" />
+    <Button displayText="0" type="digit zero" />
+    <Button displayText="." type="other" />
+    <Button displayText="=" type="operator" />
+  </div>
+);
+export default Calculator;
```
Comment (Whoistolu) [inline src/components/Calculator.js:29]: - [ ] [REQUIRED] Kindly ensure that the `Calculator.js`  component is class-based as stated in the project requirement.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,10 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (Whoistolu) [inline src/App.js:10]: - [ ] [REQUIRED] Also, kindly ensure that the `App.js` component is class-based as stated in the project requirement.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @MahdiAghaali 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by cynthiainga — APPROVED** [review]
## STATUS APPROVED ✔️ ✔️ 

Hi @MahdiAghaali, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- [ ] No linters errors ✔️ 
- [ ] Descriptive PR ✔️ 
- [ ] Good commit history ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**